ros1_bridge/__init__.py

```diff
@@ -175,6 +199,43 @@ def get_ros2_messages():
     return pkgs, msgs, rules
 
 
+def get_ros1_services(rospack=None):
+    if not rospack:
+        rospack = rospkg.RosPack()
+    srvs = []
+    pkgs = sorted([x for x in rosmsg.iterate_packages(rospack, rosmsg.MODE_SRV)])
+    for package_name, path in pkgs:
+        for message_name in rosmsg._list_types(path, 'srv', rosmsg.MODE_SRV):
+            srvs.append(Message(package_name, message_name, path))
+    return srvs
+
+
+def get_ros2_services():
+    pkgs = []
+    srvs = []
+    rules = []
+    resource_type = 'rosidl_interfaces'
+    resources = ament_index_python.get_resources(resource_type)
+    for package_name, prefix_path in resources.items():
+        pkgs.append(package_name)
+        resource, _ = ament_index_python.get_resource(resource_type, package_name)
+        interfaces = resource.splitlines()
+        service_names = [i[:-4] for i in interfaces if i.endswith('.srv')]
+        for service_name in service_names:
+            srvs.append(Message(package_name, service_name, prefix_path))
+        # check package manifest for mapping rules
+        package_path = os.path.join(prefix_path, 'share', package_name)
+        pkg = parse_package(package_path)
+        for export in pkg.exports:
+            if export.tagname != 'ros1_bridge':
+                continue
+            if 'service_mapping_rules' not in export.attributes:
```

Comment:
Please avoid a second different export for these mappings. They should be specified in a single mapping file together with the message mapping.